src/test/java/BaseTest.java

```diff
@@ -10,4 +29,75 @@ public static void chromeConfigs() {
         }
     }
 
+    public static void clickSubmitBtn() {
+        WebElement loginButton= BaseTest.driver.findElement(By.cssSelector("button[type='submit']"));
+        loginButton.click();
+    }
+
+    public static void providePassword(String password) {
+        WebElement passwordField = BaseTest.driver.findElement(By.cssSelector("[type = 'password']"));
+        passwordField.click();
+        passwordField.sendKeys(password);
+    }
+
+    public static void provideEmail(String email) {
+        WebElement emailField = BaseTest.driver.findElement(By.cssSelector("[type = 'email']"));
+        emailField.click();
+        emailField.sendKeys(email);
+    }
+
+    @AfterMethod
+    public static void tearDownBrowser() {
+        driver.quit();
+    }
+
+    protected static void validation() throws InterruptedException {
+
+        WebElement myPlaylist = driver.findElement(By.xpath("//*[@id=\"playlists\"]/ul/li[3]/a"));
```

Comment:
Recommendation: absolute XPath is not recommended for any test automation the reason is the absolute XPath starts from the root of the DOM Structure/Tree document.

![image](https://user-images.githubusercontent.com/104761482/197736327-e85bc3fb-f243-48fe-8d48-701411b2a239.png)

Use this locator:  <a> elements with an href containing "'playlist/24815" 

a[href*="playlist/24815"]`


` WebElement myPlaylist = driver.findElement(By.cssSelector("a[href*='playlist/24815']"));`